src/app/campaign/store/actions/campaign-action.service.ts

```diff
@@ -31,6 +31,8 @@ export class CampaignActionService implements OnDestroy {
     map(this.transformFlightForm),
     // if preview params changed, dispatch loadFlightPreview
     withLatestFrom(this.store.pipe(select(selectRoutedFlight))),
+    // re-using the flight form, so check that ids match
+    filter(([formState, flightState]) => !flightState || !flightState.localFlight || formState.flight.id === flightState.localFlight.id),
```

Comment:
I'm not sure I ever noticed this error... but seems like a good sanity check! +2.

Reply:
I only caught it a few times, and it would get overwritten by a subsequent update, but it seemed like a good one to catch for sure!